### DISCUSSION TITLE
docs/overview/introduction

### DISCUSSION THREAD

**Discussion by giscus** [maximikos/indeco#6] [Giscus]
# docs/overview/introduction

https://isie-platform.netlify.app/docs/overview/introduction

**Comment by maximikos** [comment]
The comment on the landing page is an *utterance*. Here, we use [giscus](https://giscus.app/), which is very similar, but differs in how the comment reaches the website owner - in this case, me. Utterances are received as [GitHub Issues](https://github.com/maximikos/indeco/issues), whereas giscus comments end up in [GitHub Discussions](https://github.com/maximikos/indeco/discussions). In addition, giscus, lets us also react to entries, see above.
